prisma/schema/user.prisma

```diff
@@ -6,14 +13,15 @@ model User {
   organization   Organization @relation(fields: [organizationId], references: [id])
   organizationId String       @map("organization_id")
 
-  role            Role
-  firstName       String           @map("first_name")
-  lastName        String           @map("last_name")
-  email           String           @unique
-  password        String
-  resetToken      String?          @map("reset_token")
-  cROrganizations CROrganization[]
-  Study           Study[]
+  role        Role
+  level       Level
+  firstName   String        @map("first_name")
+  lastName    String        @map("last_name")
+  email       String        @unique
+  password    String
+  resetToken  String?       @map("reset_token")
+  Study       Study[]
```

Comment:
Je ne suis pas sûr de comprendre pourquoi un User a toujours une liste  de Study, si on associe les études aux organisations plutôt qu'aux utilisateurs (et sachant qu'il y a maintenant UserOnStudy)

Reply:
c'est obligatoire si tu as un UserOnStudy (many to many) d'avoir des study dans user et des user dans study

Reply:
pardon, le study ici donne sur les studies que l'utilisateur a crée, j'ai renommé ca pour plus de clarté